Gems/Terrain/Code/Source/TerrainSystem/TerrainSystem.cpp

```diff
@@ -70,6 +72,11 @@ TerrainSystem::~TerrainSystem()
     Deactivate();
 }
 
+void TerrainSystem::Reflect(AZ::ReflectContext* context)
+{
+    ClipmapGenerationPassData::Reflect(context);
```

Comment:
This should get called from TerrainFeatureProcessor::Reflect(), not TerrainSystem::Reflect().  The Terrain System doesn't have any knowledge of the renderer right now.